README.md

```diff
@@ -90,6 +90,13 @@ expander.addEventListener('text-expander-value', function(event) {
 })
 ```
 
+**`text-expander-dismiss`** is an event the text-expander-element is listening to. The event can be fired in order to hide the menu provided through `provide` method. It is not expecting any parameter. It is used internally to hide the menu when Escape key it hit.
+
+```js
+const expander = document.querySelector('text-expander')
+expander.dispatchEvent('text-expander-dismiss')
+
```

Comment:
If you have the element in reference, you can call `dismiss()`, as it is part of the public API of `text-expander`.

I'm not sure we need to use events for this mechanism.

Reply:
I think I'm a bit confused here. `text-expander` element seems to only have `.keys` as public interface https://github.com/github/text-expander-element/blob/760da4af27410c93a9cf3e7ad782d77a01987a34/src/text-expander-element.ts#L210-L233 . I agree though, that having something like `.dismiss()` can be helpful. Do you suggest it instead of the event?

Reply:
I checked with @keithamus and confirmed this ^ I had the same confusion. Added dismiss() and updated the test (tks Keith for the suggestion)